app/handler.go

```diff
@@ -0,0 +1,42 @@
+package app
+
+import (
+	"go/build"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	
+	"github.com/gorilla/mux"
+	"github.com/hackedu/maestro/router"
+)
+
+var (
+	// StaticDir is the directory containing static assets.
+	StaticDir = filepath.Join(defaultBase("github.com/hackedu/maestro/app"), "static")
```

Comment:
What happens if `GOPATH` is not set? Say on a production machine where we just deploy a static binary?